js/wide-tables.jquery.js

```diff
@@ -132,6 +132,23 @@ $.fn.wideTables = function wideTablesPlugin(options) {
     $container.data('id7.wide-tables', wideTables);
   }
 
+  if (!(navigator.userAgent.indexOf('Safari') !== -1 && navigator.userAgent.indexOf('Chrome') === -1)) {
+    if (navigator.platform === 'MacIntel') {
+      // on a mac we want to show the scroll bar as we cannot detect system preferences setting
+      // for showing/hiding scroll bar.
+      // when mac users have "AUTOMATICALLY show scroll bars" set, the grid would not be able to
+      // horizontally scroll at all in non-safari browser(even with magic trackpad).
+      $('.table-responsive').css('overflow-x', 'scroll');
+      $('body').on('table-responsive-non-safari-mac-post-init', (e, callback) => {
```

Comment:
What fires this? Or am I missing something

Reply:
whoever uses this, e.g. in tabula i might do something like:

```
      $('body').trigger('table-responsive-non-safari-mac-post-init', function () {
        $('.doubleScroll-scroll-wrapper').hide();
      });
```

Reply:
```.doubleScroll-scroll-wrapper```
is a tabula thing..

Reply:
Makes sense, maybe we could mention it somewhere in the docs

Reply:
sure, i will update the doc.

Reply:
DoubleScroll is included in ID7 (the source is [here](https://github.com/UniversityofWarwick/jqDoubleScroll/blob/master/jquery.doubleScroll.js)) – should we hide the scroll wrapper directly here?

Reply:
i _think_, in normal case, it is totally fine to have it, it became an issue in tabula only because tabula moved it to a different place

Reply:
> `.doubleScroll-scroll-wrapper`
> is a tabula thing..

as @acursley pointed out, this is not technically a tabula thing :)